contracts/test/v0.1/vrf/L2VRFConsumerMock.cjs

```diff
@@ -0,0 +1,73 @@
+const { expect } = require('chai')
+const { ethers } = require('hardhat')
+const { time, loadFixture } = require('@nomicfoundation/hardhat-network-helpers')
+
+const { createSigners } = require('../utils.cjs')
+const { vrfConfig } = require('./VRFCoordinator.config.cjs')
+
+async function deploy() {
+  const {
+    account0: deployerSigner,
+    account1: consumerSigner,
+    account2,
```

Comment:
Accounts from `account2` to `account5` does not seem to be used. Let's remove it.